patches/server/0013-Paper-Plugins.patch

```diff
@@ -4712,14 +4745,15 @@ index 0000000000000000000000000000000000000000..cfc34bbb32d5cf19dbb100003a4aae62
 +     */
 +    private Path checkUpdate(Path file) throws Exception {
 +        PluginInitializerManager pluginSystem = PluginInitializerManager.instance();
-+        if (!Files.isDirectory(pluginSystem.pluginUpdatePath())) {
++        Path updateDirectory = pluginSystem.pluginUpdatePath();
++        if (updateDirectory == null || !Files.isDirectory(updateDirectory)) {
```

Comment:
Technically we don't need to check if it's a dir here anymore, right? Guess it doesn't matter

Reply:
Wanted to check here *just* incase this is changed while the server is still running.